### PR TITLE
Fix Issue 16206 - traits getOverloads fails when one of the overload is a templatized function

### PR DESCRIPTION
**Pull request by Biotronic**
Spec change for https://github.com/dlang/dmd/pull/8195

### CODE REVIEW

File: spec/traits.dd [hunk 1]
```diff
@@ -659,6 +659,7 @@ $(H2 $(GNAME getOverloads))
         $(P The first argument is an aggregate (e.g. struct/class/module).
         The second argument is a string that matches the name of
         one of the functions in that aggregate.
+        The third argument is a bool, and is optional.
```
Comment (JinShil) [inline spec/traits.dd]: Please add an explanation of what the 3rd argument does.

File: spec/traits.dd [hunk 2]
```diff
@@ -658,8 +658,10 @@ $(H2 $(GNAME getOverloads))
 
         $(P The first argument is an aggregate (e.g. struct/class/module).
         The second argument is a string that matches the name of
-        one of the functions in that aggregate.
+        one of the members in that aggregate.
+        The third argument is a bool, and is optional.
         The result is a tuple of all the overloads of that function.
+        If the third member evaluates to true, the result will include all members that matches the supplied name.
```
Comment (JinShil) [inline spec/traits.dd]: I recommend swapping this line with the one above it so the "The result..." is the last in the paragraph, and the explanation of the 3rd argument is not fragmented.

Comment (RazvanN7) [reply spec/traits.dd]: Also, "the result will include all members that **match** the supplied name"

**Comment by wilzbach** [comment]
CircleCi failure is unrelated and sorry for the unexpected problems.
The fix and explanation is here -> https://github.com/dlang/dlang.org/pull/2371

**Pull request merged**